Quran/ms.lproj/Suras.strings

```diff
@@ -0,0 +1,114 @@
+"sura_names[0]" = "Al-Fatihah";
```

Comment:
If there are no changes than the base file, then let's not have this file.

Reply:
I have updated the name of the surahs based on its Malay localizations.